content/en/guides/directory-structure/store.md

```diff
@@ -210,6 +210,38 @@ export default {
 }
 ```
 
+A complex store setup file/folder structure might look as follows (in `store/`)
+
+```
+ store
+   |_modules
+     |_shop
+       |_cart
+         |_actions.js
+         |_getters.js
+         |_mutations.js
+         |_state.js
+       |_products
+         |_itemsGroup1
+           |_actions.js
+           |_getters.js
+           |_mutations.js
+           |_state.js
+         |_itemsGroup2
+           |_actions.js
+           ...
+           |_state.js
+         |_actions.js
+         |_getters.js
+         |_mutations.js
+         |_state.js
+   |_index.js (containing plugins and other additional elements, see below)
+   |_actions.js (being the root actions, contains nuxtServerInit)
+   |_getters.js (being the root getters)
+   |_mutations.js (being the root mutations)
+   |_state.js (being the root state)
+```
```

Comment:
@RobSteward No, you don't need a `modules` directory, and I think this could be simplified a bit and still get the point across. Something like:
```
store/
--| index.js
--| ui.js
--| shop/
----| cart/
------| actions.js
------| getters.js
------| mutations.js
------| state.js
----| products/
------| mutations.js
------| state.js
------| itemsGroup1/
--------| state.js
```

Reply:
Agreed. Good enough.